spec/factories/course_wiki_namespaces.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :course_wiki_namespaces do
+    namespace { 1 }
+    courses_wikis { nil }
```

Comment:
Maybe this is related to InvalidForeignKey problem? There shouldn't be any CourseWikiNamespaces records that don't have a CoursesWikis parent record, correct?

Reply:
Not sure about this. This is just a spec file definition, can it affect actual database processes? And, the issue gets resolved when I use `foregin_key : { on_delete: cascade }` in migration file. May be I am missing something or doing something wrong.

Reply:
I think I misunderstood; I thought this PR still had that InvalidForeignKey error you mentioned. If the current migration in PR seems to work properly, that's fine.